pmoves/services/publisher-discord/main.py

```
@@ -64,6 +64,21 @@
+def _extract_webhook_domain(webhook_url: str) -> str:
+    """Extract domain from Discord webhook URL for logging (sanitizes credentials)."""
+    # Discord webhook URLs are: https://discord.com/api/webhooks/<id>/<token>
+    # Extract just the domain for logging
+    try:
+        if "discord.com" in webhook_url:
+            return "discord.com"
+        elif "discord.gg" in webhook_url:
+            return "discord.gg"
+        else:
+            return "unknown"
+    except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)